### PR TITLE
Use irc-framework setTopic() for topic command

### PR DESCRIPTION
**Pull request by MaxLeiter**
Possible due to #1070

### CODE REVIEW

File: src/plugins/inputs/topic.js [hunk 1]
```diff
@@ -14,9 +15,6 @@ exports.input = function(network, chan, cmd, args) {
 
 		return;
 	}
-
-	var irc = network.irc;
-	irc.raw("TOPIC", chan.name, args.join(" "));
-
+	client.setTopic(chan.name, args.join(" "));
```
Comment (astorije) [inline src/plugins/inputs/topic.js]: Why not use `this`?

Comment (MaxLeiter) [inline src/plugins/inputs/topic.js]: client is more clear imo, also consistent  with other files, but nbd to change

Comment (astorije) [inline src/plugins/inputs/topic.js]: Need to change on L10/R11 as well then. I'll approve after that and let @xPaw decide if he cares or not.

Comment (xPaw) [inline src/plugins/inputs/topic.js]: It should be `network.irc` though.

**Comment by MaxLeiter** [comment]
Actually got on the laptop and fixed it. Good now.

**Comment by xPaw** [comment]
Did you test that `/topic` still retrieves the topic?

**Comment by MaxLeiter** [comment]
Still works, @xPaw

**Pull request merged**